lib/ship.rb

```diff
@@ -0,0 +1,22 @@
+class Ship
+  attr_reader :name, :length, :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
+    @health = 3
+    @sunk = false
+  end
+
+  def sunk?
+    if @health == 0
+      @sunk = true
+    else
+      @sunk
+    end
+  end
+
+  def hit
```

Comment:
We need an `if` statement to only subtract health `if health > 0` so we don't ever get a negative value for health